libraries/anvill_passes/src/RecoverStackFrameInformation.cpp

```diff
@@ -65,8 +65,9 @@ bool RecoverStackFrameInformation::Run(llvm::Function &function) {
   // It is now time to patch the function. This method will take the stack
   // analysis and use it to generate a stack frame type and update all the
   // store and load instructions
-  auto update_func_res = UpdateFunction(
-      function, stack_frame_analysis, options.zero_init_recovered_stack_frames);
+  auto update_func_res =
+      UpdateFunction(function, stack_frame_analysis,
+                     options.stack_frame_struct_init_procedure);
 
   if (!update_func_res.Succeeded()) {
     auto error_code =
```

Comment:
If this is going to be frequent, then can you make an `ErrorString` or `ErrorCodeString` method that does this stringifying?